### DISCUSSION TITLE
VDH ver. 8.2.2.8 for CHROME with WINOWS 11 ***CANNOT DOWNLOAD TWITCH CLIP***

### DISCUSSION THREAD

**Discussion by romelussaladar** [aclap-dev/video-downloadhelper#313] [Help and Q&A]
### Your question

Hello...  I just tried to download a clip on twitch and it keeps showing ERROR messages. I attempted to close chrome to update it, and turn on VDH again several times, and continuously get the following error message.

**Error in invocation of downloads.download(downloads.DownloadOptions options, optional function callback): Error at parameter 'options': Unexpected property: 'incognito'.
TypeError: Error in invocation of downloads.download(downloads.DownloadOptions options, optional function callback): Error at parameter 'options': Unexpected property: 'incognito'.
    at chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:7834
    at new Promise (<anonymous>)
    at Proxy.<anonymous> (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:7515)
    at Object.apply (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:7916)
    at R2 (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:9777)
    at async ri (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:13229)**
    
previously I could download all clips 4 days ago.

### Addon & CoApp version and details

Current info for CHROME Chrome is up to date
Version 122.0.6261.129 (Official Build) (64-bit)

WINDOWS 11 home ver 23H2

VDH ver updated 8.2.2.8

**Comment by GrampaWildWilly** [comment]
Search this forum using "incognito" as your search key.

![image](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/3703e43d-477d-4e7e-9b91-1b26ad220883)

Do not use the browser's search field at the top of the page.  Use the search field within this web page, the one marked "Type to search" in the upper right portion of the web page.  That's right below the browser search field.

![image](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/95d5be77-5311-4c1c-bfdd-c68c6ea1e0c6)

**Reply by romelussaladar** [reply]
Hello @GrampaWildWilly ,

I didnt need to do anything. Recent VDH update 8.2.2.9 now downloading same old clip that did not work in the previous day.

Appears to be working.

REGARDS,

Rom all GOOD !!!
